docs/release-notes.rst

```diff
@@ -6,10 +6,91 @@
  Release Notes
 ###############
 
+**************
+ Version 0.33
+**************
+
+Version 0.33.0
+==============
+
+**Release Date:** May 29, 2024
+
+**Breaking Changes**
+
+-  Helm: An entry for ``initialUserPassword`` is now required when running ``helm install``.
+   Existing deployments are unaffected. See :ref:`Helm Chart <helm-config-reference>`.
+
+-  Web UI: Enforce password requirements for all new non-remote users: see
+   :ref:`password-requirements`
+
+   -  This applies to users created using the ``Add User`` button in the Web UI for admins.
+
+   -  Admins may also change the passwords of other users with this same interface.
+
+   -  This does not affect logging in with an existing user who already has an empty or
+      non-compliant password, but we recommend setting good passwords for such users as soon as
+      possible.
+
```

Comment:
- **WebUI**: Enforce password requirements for all new non-remote users. See :ref:`password-requirements` for details.

   - Applies to users created using the **Add User** button in the WebUI for admins.
   - Admins can change the passwords of other users using the same interface.
   - Does not affect existing users with empty or non-compliant passwords, but setting strong passwords for these users is recommended.

Reply:
Updated